hpat/hiframes/series_kernels.py

```diff
@@ -492,7 +492,6 @@ def gt_f(a, b):
 
 
 series_replace_funcs = {
-    'sum': _column_sum_impl_basic,
```

Comment:
please, don't delete it - keep it as comments. It might be helpful for us to see this code in short term. We delete it a bit later